### ISSUE TITLE
Feature request: Support the “N-th” occurrence of a weekday of the month

### ISSUE THREAD

**Issue by ssd3** [Hexagon/croner#198]
**Describe the bug**
I need to have the scheduled event for example 
At 10 a.m. on the third Monday of every month: 0 10 ? * 2#3

code sample:
```
startCron("Job 1", {
	what: "Job data 1"
}, "0 10 ? * 2#3", "America/New_York");
```

Error:
TypeError: CronPattern: configuration entry 5 (2#3) contains illegal characters.

**Comment by Hexagon** [comment]
It is possible right now, by setting legacyMode to false and supplying both day of month and weekday, such as * * * 1-7 * SUN for first sunday of month, or 15-21 * SUN for third sunday of month. But I'll keep this as a feature request.

**Comment by Hexagon** [comment]
Oh, there is a feature request in discussions already.

https://github.com/Hexagon/croner/discussions/190

**Comment by Hexagon** [comment]
Re-opening as this is work in progress

**Comment by Hexagon** [comment]
Feature pre-released in `7.0.0-dev.0`

Install using `npm i croner@dev`

**Comment by Hexagon** [comment]
Released in `7.0.0`
